frame/elections-phragmen/src/lib.rs

```diff
@@ -878,6 +879,13 @@ impl<T: Trait> Contains<T::AccountId> for Module<T> {
 	}
 }
 
+/// Implementation uses a parameter type so calling is cost-free.
+impl<T: Trait> ContainsCountUpperBound for Module<T> {
+	fn count_upper_bound() -> usize {
```

Comment:
suggestion: rename the Trait to `CountUpperBound` or just `UpperBound` and the function to `upper_bound`

Reply:
yes but then implementing an upper bound for the module election_phragmen looks weird, the module doesn't have any upper bond, it contains something which have a limit on its number of elements.

Maybe ContainsUpperBound. but as Kian suggested I renamed ContainsLengthBound.